medallion/test/test_startup.py

```diff
@@ -0,0 +1,282 @@
+from unittest import mock
+
+import pytest
+import pytest_subtests  # noqa: F401
+
+import medallion.config
+import medallion.scripts.run
+
+
+def test_parser_help(capsys):
+    """
+    Confirm that the parser help can be printed with the custom formatter.
+    """
+    parser = medallion.scripts.run._get_argparser()
+    parser.print_help()
+    (out, _) = capsys.readouterr()
+    assert "medallion v" in out
```

Comment:
Was this supposed to be the whole text of the help command? I assume "medallion v" is just a random section of it, and that's okay, just seemed odd.

Reply:
It's just a reliable looking substring from the help text. The current version number follows the `v` at the end here and the test shouldn't care about that. It's mostly to validate that the custom formatter is doing its thing :)